README.md

```diff
@@ -1,54 +1,82 @@
 
-```shell
-go get github.com/IBM-Cloud/power-go-client
-```
+## Prerequisites
+
+- An [IBM Cloud Account](https://cloud.ibm.com/registration)
+- An IBM Cloud [IAM API key](https://cloud.ibm.com/docs/account?topic=account-userapikey) or [Token](https://cloud.ibm.com/docs/account?topic=account-iamtoken_from_apikey) allow the SDK to access your account.
+- Go version 1.17 or above.
+
+## Install
+
+Install the SDK using the following methods.
 
-Update the SDK to the latest version using the following command
+### `go get` command
 
-```shell
+Use this command to download and install the SDK to allow your Go application to use it:
+```
 go get -u github.com/IBM-Cloud/power-go-client
 ```
 
-Using the SDK
+### Go modules
+
+If your application is using Go modules, you can add a suitable import to your Go application, like this:
+```go
+import (
+  "github.com/IBM-Cloud/power-go-client"
+)
+```
+then run `go mod tidy` to download and install the new dependency and update your Go application's
+`go.mod` file.
 
-You must have a working IBM Cloud account to use the APIs. Sign up if you don't have one.
+### `dep` dependency manager
 
-The SDK has examples folder which cites few examples on how to use the SDK. First you need to create a session.
+If your application is using the `dep` dependency management tool, you can add a dependency
+to your `Gopkg.toml` file. Here is an example:
+```
+[[constraint]]
+  name = "github.com/IBM-Cloud/power-go-client"
+  version = "1.1.0"
+
+```
+then run `dep ensure`.
+
+## Usage
+
+The SDK has [examples](examples) directory that shows how to use the SDK. First you need to create a session and use it for creating the client.
 
 ```golang
 import "github.com/IBM-Cloud/power-go-client/ibmpisession"
 
 func main(){
-
-    s := ibmpowersession.New()
+    o := &ibmpisession.IBMPIOptions{
+		Authenticator: authenticator,
+		UserAccount:   accountID,
+		Zone:          zone,
+		URL:           url,
+	}
+    s, err := ibmpisession.NewIBMPISession(o)
     .....
 }
 ```
+- `authenticator`: Please check https://github.com/IBM/go-sdk-core/blob/main/Authentication.md for different options available for authenticating API calls.
+- `accountID`: Account id of the Power Cloud Service Instance
+- `zone`: Zone of the Power Cloud Service Instance
+- `url`: Power Virtual Server host or URL endpoint. You can also set env variable `IBMCLOUD_POWER_API_ENDPOINT`.
 
-Creating session in this way creates a default configuration which reads the value from the environment variables. You must export the following environment variables.
-
-    IBMID - This is the IBM ID
-    IBMID_PASSWORD - This is the password for the above ID
-
-OR
+## Issues
 
-    IC_API_KEY/IBMCLOUD_API_KEY - This is the Bluemix API Key. Login to IBMCloud to create one if you don't already have one. See instructions below for creating an API Key.
+If you encounter an issue with the project please [report here.](https://github.com/IBM/networking-go-sdk/issues).
```

Comment:
Is this the correct URL to report problems?

Reply:
opps.. copy paste mistake 👎